machine_tool_rental/demo/sale_order_line.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="sale_order_line_6" model="sale.order.line">
+        <field name="order_id" ref="sale_order_6"/>
+        <field name="product_id" ref="product_product_9"/>
+        <field name="is_rental" eval="True"/>
+        <field name="name">
+            <![CDATA[ Excavator 300.9D 7/15/25 15:00 to 9/15/25 15:00 ]]>
+        </field>
+        <field name="product_uom_id" ref="uom.product_uom_unit"/>
+        <field name="price_unit">7750.0</field>
+        <field name="technical_price_unit">7750.0</field>
```

Comment:
This is probably computed (and the period in the name will be wrong). Could you test?
```suggestion
```